woocommerce/woocommerce-ios
# Add Buildkite Release Builds

Moves Release Builds to Buildkite.

**To Test:**
- Ensure that all tests pass
- Note that https://buildkite.com/automattic/woocommerce-ios/builds/3494 fails on the `upload` phase due to a duplicate binary.
